### PR TITLE
Added **kwargs for embedding funcs

### PR DESCRIPTION
**Pull request by FrancescoSaverioZuppichini**
Hi There,

This is a simple PR, adding **kwargs to `.embed_query` similar to the other OpenAI functions. This is handy because I can add to the request whatever I want, in my case I am running OpenAI behind a custom proxy and I need a couple of more things to the requests

@baskaryan

Thanks

Cheers, 

Fra

### CODE REVIEW

File: langchain/embeddings/openai.py [hunk 1]
```diff
@@ -461,7 +463,7 @@ async def _aembedding_func(self, text: str, *, engine: str) -> List[float]:
             )["data"][0]["embedding"]
 
     def embed_documents(
-        self, texts: List[str], chunk_size: Optional[int] = 0
+        self, texts: List[str], chunk_size: Optional[int] = 0, **kwargs
```
Comment (baskaryan) [inline langchain/embeddings/openai.py:466]: this signature is defined on base embeddings class, probably don't want to break the interface just here and im not sure we should add it to the base class.

what if added something like this instead? https://github.com/hwchase17/langchain/blob/c7b687e944883df972cabdf00064112587306daf/langchain/llms/openai.py#L137

Comment (baskaryan) [reply langchain/embeddings/openai.py:466]: for embeddings you can already pass model in directly, no? 
```python
OpenAIEmbeddings(model=...)
```

Comment (FrancescoSaverioZuppichini) [reply langchain/embeddings/openai.py:466]: yes but I need to place it also on the request body because that is parsed by our custom proxy 😢

**Comment by leo-gan** [comment]
@FrancescoSaverioZuppichini Hi , could you, please, resolve the merging issues and address the last comments (if needed)? After that, ping me and I push this PR for the review. Thanks!

**Comment by FrancescoSaverioZuppichini** [comment]
> @FrancescoSaverioZuppichini Hi , could you, please, resolve the merging issues and address the last comments (if needed)? After that, ping me and I push this PR for the review. Thanks!

sorry, no time

**Comment by leo-gan** [comment]
@baskaryan I'm closing it.

**Pull request closed without merging**